src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackGeneralSettings.qml

```diff
@@ -47,62 +47,94 @@ Item {
 
         spacing: 12
 
-        NoteExpandableBlank {
-            id: noteSection
-            navigation.panel: root.navigationPanel
-            navigation.row: root.navigationRowStart
-
-            model: proxyModel ? proxyModel.modelByType(Inspector.TYPE_NOTE) : null
+        readonly property var fullModel: [
+            { typeRole: Inspector.TYPE_NOTE, componentRole: noteSection },
+            { typeRole: Inspector.TYPE_ARPEGGIO, componentRole: arpeggioSection },
+            { typeRole: Inspector.TYPE_FERMATA, componentRole: fermataSection },
+            { typeRole: Inspector.TYPE_BREATH, componentRole: pausesSection },
+            { typeRole: Inspector.TYPE_GLISSANDO, componentRole: glissandoSection },
+            { typeRole: Inspector.TYPE_GRADUAL_TEMPO_CHANGE, componentRole: tempoChangeSection }
+        ]
+
+        function getVisibleModel() {
+            var visibleModel = []
+            for (let i = 0; i < fullModel.length; i++) {
+                var itemModel = root.proxyModel ? root.proxyModel.modelByType(fullModel[i].typeRole) : null
+                if (Boolean(itemModel) && !itemModel.isEmpty) {
+                    fullModel[i].itemModel = itemModel
```

Comment:
Just one moment - you are changing readonly property (no qml warning in console?)
May be it is better to push the object like this to visibleModel?
`visibleModel.push({ component: fullModel[i].componentRole, model: itemModel)`